### ISSUE TITLE
Norway problem in yaml output from xkbcli list

### ISSUE THREAD

**Issue by hickford** [xkbcommon/libxkbcommon#268]
> Anyone who uses YAML long enough will eventually get burned when attempting to abbreviate Norway.

https://news.ycombinator.com/item?id=17359376

YAML output from `xkbcli list`

```yaml
- layout: no
  variant: ''
  brief: no
  description: Norwegian
  iso639: [nor, nob, nno]
  iso3166: []
```

Output of PyYAML's `yaml.safe_load`

```python
{'layout': False,
 'variant': '',
 'brief': False,
 'description': 'Norwegian',
 'iso639': ['nor', 'nob', 'nno'],
 'iso3166': []},
```

**Comment by hickford** [comment]
This is easiest solved by more quoting

**Comment by hickford** [comment]
Workaround: parse with `strictyaml.dirty_load( , allow_flow_style=True)`

**Comment by hickford** [comment]
Turns out this is a Yaml 1.1 problem fixed in 1.2.  PyYAML only supports the older https://github.com/yaml/pyyaml/issues/116

Better to use parser ruamel.yaml https://yaml.readthedocs.io/en/latest/

**Comment by bluetech** [comment]
Heh that's a funny problem. I've made a change to mention YAML 1.2 in the manpage: e8cb431144f8bceb0ad2443cb07b383dcbed9280
